.circleci/config.yml

```
@@ -89,7 +89,7 @@ jobs:
               name: Deploy docs to gh-pages branch
               command: |
                 touch doc/_build/html/.nojekyll
-                gh-pages --dotfiles --message "doc updates [skip ci]" --dist doc/_build/html --dest .
+                gh-pages --dotfiles --message "doc updates [skip ci]" --dist doc/_build/html --dest ./dev
```

```suggestion
                gh-pages --dotfiles --message "doc updates [skip ci]" --dist doc/_build/html --dest ./stable
                gh-pages --dotfiles --message "doc updates [skip ci]" --dist doc/_build/html --dest ./v0.1
```

so the docs get pushed to the right folder